11ty/definitions/blacklist-whitelist.md

```diff
@@ -0,0 +1,63 @@
+---
+title: blacklist whitelist
+slug: blacklist-whitelist
+flag:
+  level: avoid
+  text: racist term
+defined: true
+excerpt: 'umbrella term used to represent a list of entities/people who are to be allowed or denied access to a service, privilege, or recognition: to "blacklist" is "to exclude", to "whitelist" means "to include".'
+speech: noun; verb
+alt_words:
+  blacklist (noun):
+    - blocklist
+    - denylist
+    - excludelist
+  whitelist (noun):
+    - allowlist
+    - passlist
+    - trustlist
+    - safelist
+  to blacklist (verb):
+    - to block
+    - to deny
+    - to exclude
+  to whitelist (verb):
+    - to allow
+    - to trust
+    - to include
+reading:
+  - text: UK security agency bans blacklist and whitelist for being racist
+    href: https://www.techspot.com/news/85080-uk-cyber-security-agency-stop-using-racist-blacklist.html
+  - text: Google Chrome team moves away from black/whitelist to be more inclusing
+    href: https://9to5google.com/2020/06/12/google-android-chrome-blacklist-blocklist-more-inclusive/
+  - text: Google developers styleguide entry for blacklist
+    href: https://developers.google.com/style/word-list#blacklist
+  - text: '"Blacklists” and “whitelists": a salutary warning concerning the prevalence of racist language in discussions of predatory publishing'
+    href: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/
+---
+
+To allow or deny access to a service, a privilege, or recognition.
+
+## Issues
+
+<!-- Black/white admittance and denial (Jim Crow) -->
+
+The first problem is the admittance and denial of people based on their race and skin color. The term are rooted in racism: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
```

Comment:
```suggestion
The first problem is the admittance and denial of people based on their race and skin color. The terms are rooted in racist history: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
```
I think here we need to be mindful of breaking down this connection a bit more. According to the research I did, `blacklist` came from the 1600s and "black books." So, while _etymologically_ the origins predate American slavery, it's that the terms black and white in *modern* usage carry a racial meaning. By using these terms, we're reinforcing the idea that black is bad and white is good, and therefore, reinforcing the systemic and infrastructural racism in a small and powerful way.

Mostly I want to ensure that "well actually" people don't "get us" on a technicality (and also I want to ensure we're promoting as much historical accuracy as possible, while still reinforcing the idea that there are better terms.)